backend/src/db/migrations/20230909170408_change_meditation_entries.ts

```diff
@@ -0,0 +1,40 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+  MEDITATION_ENTRIES: 'meditation_entries',
+  MEDITATION_TOPICS: 'meditation_topics',
+};
+
+const ColumnName = {
+  ID: 'id',
+  TOPIC_ID: 'topic_id',
+  AUDIO_URL: 'audio_url',
+} as const;
+
+type MeditationTopic = {
+  id: number;
+};
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TableName.MEDITATION_ENTRIES, (table) => {
+    table.integer(ColumnName.TOPIC_ID).nullable().alter();
+    table.text(ColumnName.AUDIO_URL).notNullable().defaultTo('');
+  });
+}
+
+async function down(knex: Knex): Promise<void> {
+  const topic = await knex(TableName.MEDITATION_TOPICS)
```

Comment:
will this work if we don't have topics and entries as well?

Reply:
@Liza-Veis Definitely no) Should I delete the record where `topic_id` is null?

Reply:
yes, let's delete such records

Reply:
also let's split this to two migrations

Reply:
Done